cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabasePrivilegeTest.scala

```diff
@@ -0,0 +1,153 @@
+
+  private def setup() = graph.withTx { tx =>
+    tx.execute("CREATE ROLE my_role")
+    tx.execute("CREATE DATABASE foo")
+    tx.execute("CREATE DATABASE bar")
+    tx.execute("CREATE DATABASE baz")
+    tx.execute("GRANT CREATE NEW PROPERTY NAMES ON DATABASE bar TO my_role")
+  }
+
+  def text: String = {
+    setup()
+    """
+###assertion=update-one
+//
+
+GRANT ACCESS ON DATABASE * TO my_role
+###
+
+Grant privilege to access and run queries against all databases to the role `my_role`.
+
+###assertion=update-one
+//
+
+GRANT START ON DATABASE * TO my_role
+###
+
+Grant privilege to start all databases to the role `my_role`.
+
+###assertion=update-one
+//
+
+GRANT STOP ON DATABASE * TO my_role
+###
+
+Grant privilege to stop all databases to the role `my_role`.
+
+###assertion=update-one
+//
+
+GRANT CREATE INDEX ON DATABASE foo TO my_role
+###
+
+Grant privilege to create indexes on the database `foo` to the role `my_role`.
+
+###assertion=update-one
+//
+
+GRANT DROP INDEX ON DATABASE foo TO my_role
+###
+
+Grant privilege to drop indexes on the database `foo` to the role `my_role`.
+
+###assertion=update-two
+//
+
+DENY INDEX MANAGEMENT ON DATABASE bar TO my_role
+###
+
+Deny privilege to create and drop indexes on the database `bar` to the role `my_role`.
+
+###assertion=update-one
+//
+
+GRANT CREATE CONSTRAINT ON DATABASE * TO my_role
+###
+
+Grant privilege to create constraints on all databases to the role `my_role`.
+
+###assertion=update-one
+//
+
+DENY DROP CONSTRAINT ON DATABASE * TO my_role
+###
+
+Deny privilege to drop constraints on all databases to the role `my_role`.
+
+###assertion=update-two
+//
+
+REVOKE CONSTRAINT ON DATABASE * FROM my_role
+###
+
+Revoke existing privileges (both granted and denied) to create and drop constraints on all databases from the role `my_role`.
+
+###assertion=update-one
+//
+
+GRANT CREATE NEW LABELS ON DATABASE * TO my_role
+###
+
+Grant privilege to create new labels on all databases to the role `my_role`.
+
+###assertion=update-one
+//
+
+DENY CREATE NEW TYPES ON DATABASE foo TO my_role
+###
+
+Deny privilege to create new relationship types on the database `foo` to the role `my_role`.
+
+###assertion=update-one
+//
+
+REVOKE GRANT CREATE NEW PROPERTY NAMES ON DATABASE bar FROM my_role
+###
+
+Revoke grant privilege to create new property keys on the database `bar` from the role `my_role`.
+
+###assertion=update-two
+//
+
+GRANT NAME MANAGEMENT ON DATABASE * TO my_role
+###
+
+Grant privilege to create labels, relationship types and property keys on all databases to the role `my_role`.
+
+###assertion=update-ten
+//
+
+GRANT ALL ON DATABASE baz TO my_role
+###
+
+Grant all database privileges on the database `baz` to the role `my_role`.
+This means that users with `my_role` can access, start and stop the database as well as
+create and drop indexes and constraints and create labels, relationship types and property keys on the database.
```

Comment:
I think it is a bit risky to list all privileges here, since we might forget to update the list if we add any new database privileges in the future. Maybe it is better to just write ``Grant all database privileges above on the database `baz` to the role `my_role` ``? Or just mention a few examples in the text. What do you think?